packages/openneuro-components/src/form/RadioGroup.tsx

```diff
@@ -0,0 +1,32 @@
+import React, { useState } from 'react'
+import { Radio } from './Radio'
+
+import './forms.scss'
+
+export interface RadioGroupProps {
+  id: string
+  layout: string
+}
+
+export const RadioGroup: React.FC<RadioGroupProps> = ({
+  radioArr,
+  id,
```

Comment:
I think we can remove `id` usage until we run into a situation where we need to bring it back